app/views/shared/_ie_fallbacks.html.slim

```diff
@@ -0,0 +1,5 @@
+- nonce = content_security_policy_script_nonce
+
+<script nonce="#{nonce}" src='https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js'></script>
```

Comment:
I think there have just been errors on IE < 9 for the lifetime of the application. I would think we would want the HTML shiv at least, to get access to html5 features...but if we don't care about those and the application has been working acceptably on IE8 the I suppose we can remove them